src/interpret/atac_marker_peaks/script.py

```diff
@@ -0,0 +1,42 @@
+import mudata
+import muon as mu
+import pandas as pd
+
+### VIASH START
+par = {
+    "input": "resources_test/pbmc_1k_protein_v3/pbmc_1k_protein_v3_mms.h5mu",
+    "output": "foo.h5mu",
+    "output_marker_peaks": "marker_peaks.tsv",
+    "output_compression": "gzip",
+    "modality": "atac",
+    "groupby": "leiden",
+    "method": "t-test",
+}
+### VIASH END
+
+def main():
+    mdata = mudata.read(par["input"].strip())
+    mod = mdata.mod[par["modality"]]
+
+    if "peak_annotation" not in mod.uns["atac"]:
```

Comment:
Could you create an argument for the location of the peaks?

Reply:
This is the default value from `muon`: https://muon.readthedocs.io/en/latest/api/generated/muon.atac.tl.add_peak_annotation.html

Other `muon` functions check exactly this location, so if we store it somewhere else, the errors will appear

Reply:
I think what we could do is add an argument here (with a default value  `peak_annotation`) and also add this argument to other components where needed?